### DISCUSSION TITLE
integrals yielding inverse trigonometric functions /sqrt problem

### DISCUSSION THREAD

**Discussion by loubaliber** [3b1b/manim#1486] [Q&A] [answered]
Hi, I need help.
I want to write integrals yielding inverse trigonometric functions in manim. 

This is my code:

```
class InverseTrig(Scene):
    def construct(self):
        eq1=Tex("\\int \\frac{du}{\sqrt{a^{2} - u^{2}}}\\ = \\ sin^{-1} (\\frac{u}{a}) + C ")
        eq1.shift(2*UP)
        eq2=Tex("\\int \\frac{du}{a^{2} + u^{2}} = \\frac{1}{a} tan^{-1} (\\frac{u}{a}) + C  ")
        eq3=Tex("\\int \\frac{du}{u \sqrt{u^{2}-a^{2}}}\\ = \\ \\frac{1}{a} sec^{-1} \\frac{u}{a} + C ")
        eq3.shift(2*DOWN)
        
        self.play(Write(eq1))
        self.wait(1)
        self.play(Write(eq2))
        self.wait(1)
        self.play(Write(eq3))
        self.wait(3)
```

But what I get is this:

![image](https://user-images.githubusercontent.com/76828693/115183798-23139780-a10f-11eb-9ce5-d3dc1f45fa62.png)

**Comment by TonyCrane** [comment, marked as answer]
Clean the cache in Tex/ and try it again?
I run your code and got the correct result.

<img width="806" alt="截屏2021-06-12 下午6 28 22" src="https://user-images.githubusercontent.com/44120331/121773080-20cc3780-cbac-11eb-8d13-0f11b03870ee.png">
